HW03/ihar_shkuryn/ihar_shkuryn_hw_03_t_01.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(log)
+    log.split("\n").select { |line| line.include?('error') } [0].to_s
```

Comment:
Could you read about the difference between `find` and `select` and write you answer in my telegram direct please?

Reply:
done